### ISSUE TITLE
Question about -ej option in macs

### ISSUE THREAD

**Issue by cdquinto** [gchen98/macs#33]
Hello again, 

When the option -ej is used, what happens to the size of population j? Is it the same as specified in the command?
In particular, I want to know if these 2 commands will have the same behavior.
Command 1:
macs 10 100000 -t 0.004 -r 0.004 -h 1e5 -I 2 5 5 -n 1 2 -n 2 2 -ej 0.0003 2 1 -en 0.000301 1 2 
-en 0.00005 1 1
Command 2: 
macs 10 100000 -t 0.004 -r 0.004 -h 1e5 -I 2 5 5 -n 1 2 -n 2 2 -ej 0.0003 2 1 -en 0.00005 1 1

Thank you

**Comment by gchen98** [comment]
For population join events all chromosomes from the population specified 
in the first argument are migrated to the population specfied in the 
second argument.

**Comment by cdquinto** [comment]
In my example,

macs 10 100000 -t 0.004 -r 0.004 -h 1e5 -I 2 5 5 -n 1 2 -n 2 2 -ej 0.0003 2
1

Population 1 has size 2 before receiving the chromosomes from population 2.

After these populations are joined, does population 1 still has size 2?

**Comment by gchen98** [comment]
Yes. It receives migrants from 2, but the population size is an 
independent parameter.
